alibi_detect/od/pytorch/ensemble.py

```diff
@@ -0,0 +1,341 @@
+from __future__ import annotations
+from abc import ABC, abstractmethod
+from typing import Optional
+
+import torch
+import numpy as np
+from torch.nn import Module
+
+from alibi_detect.base import NotFitException
+
+
+class BaseTransformTorch(Module, ABC):
```

Comment:
Because the public API methods `fit` and `transform` are inherited from the two base classes, what does it do to our public docs of these methods? Are the descriptions of the base `fit` and `transform` methods enough to be shared across all implementations of transformations?

Reply:
E.g. with the current setup it seems we may be unable to document what is returned by a specific transormations `transform` method (1-p-vals for or mean of top-k scores etc.).

Reply:
I've removed the private fit and transform patterns as I couldn't figure out how to document them otherwise.